constants.py

```diff
@@ -6,6 +6,9 @@
     "storage": {"appdata": "%APPDATA%/TFT Bot"},
     "executables": {
         "league": {
+            "client_base": r"\LeagueClient.exe",
+            "client_ux_base": r"\LeagueClientUx.exe",
+            "game_base": r"\Game\League of Legends.exe",
             "client": r"\LeagueClient.exe",
             "client_ux": r"\LeagueClientUx.exe",
             "game": r"\Game\League of Legends.exe",
```

Comment:
Works for me -- though would you think it might make sense to have the existing values have a different value until they are assigned the first time, since they are never read unless they are written, first? 🤔 (just to avoid a future bug)
```suggestion
            "client": r"\unassigned_client_exe",
            "client_ux": r"\unassigned_client_ui_exe",
            "game": r"\unassigned_game_exe",
```

Reply:
Doesn't make a big difference. Either way will fail if it is ever read before being updated.

Reply:
Oh I see what you're saying, yeah since it needs the base directory to be pre-pended, yeah that works for me 👌